addons/edi_sale/models/edi_sale_request_document.py

```diff
@@ -95,13 +95,32 @@ def prepare(self, doc):
     @api.model
     def execute(self, doc):
         """Execute document"""
+        Partner = self.env['res.partner']
+
         super().execute(doc)
 
+        # Delete sale orders which no longer have any order lines.
+        SaleRequestRecord = self.sale_request_record_model(doc)
```

Comment:
`SaleRequestRecord` and `Partner` should be in the same place, before or after `super()`.

Reply:
Done, please review.